ui/src/pages/Questions/Detail/components/Reactions/index.tsx

```diff
@@ -0,0 +1,146 @@
+import { FC, memo, useEffect, useState } from 'react';
+import { Button, OverlayTrigger, Tooltip } from 'react-bootstrap';
+import { useTranslation } from 'react-i18next';
+
+import classNames from 'classnames';
+
+import { Icon } from '@/components';
+import { queryReactions, updateReaction } from '@/services';
+import { tryNormalLogged } from '@/utils/guard';
+import { ReactionItem } from '@/common/interface';
+
+interface Props {
+  objectId: string;
+  showAddCommentBtn?: boolean;
+  handleClickComment: () => void;
+}
+
+const emojiMap = [
+  {
+    name: 'heart',
+    icon: 'heart-fill',
+    className: 'text-danger',
+  },
+  {
+    name: 'smile',
+    icon: 'emoji-laughing-fill',
+    className: 'text-warning',
+  },
+  {
+    name: 'frown',
+    icon: 'emoji-frown-fill',
+    className: 'text-warning',
+  },
+];
+
+const Index: FC<Props> = ({
+  objectId,
+  showAddCommentBtn,
+  handleClickComment,
+}) => {
+  const [reactions, setReactions] = useState<Record<string, ReactionItem>>();
```

Comment:
Add this code so that the add emoji button remains active

```
const [reactIsActive, setReactIsActive] = useState(false);
```

Reply:
resolved.